main/2021/CG-08-03.md

```diff
@@ -30,12 +30,12 @@ Installation is required, see the calendar invite.
     1. Summary of [interface-types/#135](https://github.com/WebAssembly/interface-types/issues/135) and discussion [40 min].
         1. Poll for supporting [UTF-16](https://github.com/WebAssembly/interface-types/issues/136#issuecomment-861799460) lifting and lowering
         1. Poll for supporting compact UTF-16 (Latin1) lifting and lowering
-        1. Discussion: Is the Component Model in-line with our high-level goals?
-           * i.e. "integrate well with the existing Web platform"
-           * i.e. "execute in the same semantic universe as JavaScript"
-           * i.e. "maintain the backwards-compatible nature of the Web"
-           * i.e. "promote other compilers and tools targeting WebAssembly"
-           * i.e. "WebAssembly **also** supports non-web embeddings" (but not primarily)
+        1. Discussion: Is the Component Model in-line with our [high-level goals](https://webassembly.org/docs/high-level-goals/), with [what](https://webassembly.org/) we [communicate](https://www.w3.org/2020/03/webassembly-wg-charter), and [envisioned](https://www.youtube.com/watch?v=ZKydJPRosa0)? In particular:
+           * "integrate well with the existing Web platform"
+           * "execute in the same semantic universe as JavaScript"
+           * "maintain the backwards-compatible nature of the Web"
+           * "promote other compilers and tools targeting WebAssembly"
+           * "WebAssembly **also** supports non-web embeddings" (but not primarily)
         1. Poll for retaining compatibility with `DOMString` - or otherwise - maintain a single list-of-USV `string` type
```

Comment:
Better, but "retaining compatibility with `DOMString`" is misleading. It's not as though introducing the component model "breaks" compatibility that was there before. Also, referring to the transmission of isolated surrogates across component boundaries as "compatibility with DOMString" severely obscures the true point of the vote.

I recommend

"Poll to either maintain a single list-of-USV `string` type, or alternatively allow isolated surrogates to cross component boundaries"

Reply:
Well, you either support `DOMString`, or not, then you are supporting `USVString`, no?

Reply:
Clarified the distinction between the two WebIDL concepts, and removed "compatibility" as per your request.

Reply:
Btw, I think a very important aspect here is the word "single" that may eventually bite us since we cannot even introduce a working alternative in the future, or for the Web embedding. Would it be OK if I remove it, or emphasize it prominently?

Reply:
I think @conrad-watt's suggested wording is good.  I understand "single" to mean "in the current proposal", so it'd also be fine to include that sort of qualifier.

Reply:
OK, if this is so important to you, I could add an "also", "as well" or similar to the alternative to indicate that we'd then decide to support both, not one. Does this sound good?

Reply:
This looks reasonable to me, but I would not include the "also." If the CG has consensus that we should support transmission of isolated surrogates, then I think we would then want to evaluate separately whether it's worth also supporting strict USV strings.

Reply:
Hmm, yeah, makes sense. I wanted to be as diplomatic as possible, as I think that this is not an either or, and as clear as possible, as people have argued in the past that I'd be advocating for just one, but not both which I'd prefer. Do you see another way to clarify this nuance (or no need to), so my perspective isn't as easily misunderstood?

Reply:
I don't think there's any need to clarify that nuance here on the agenda, but I'm sure it will come up in discussion and we can make sure it is reflected in the notes.

Reply:
I don't think it's correct to conflate "strings with lone surrogates" with `DOMString` (by saying "aka `DOMString`") because `DOMString` also generally suggests an *encoding* (WTF-16).  As we keep saying in the other issue, this is a poll about string *semantics* so please use exactly the text @conrad-watt proposed.

Reply:
What if I say "`DOMString` semantics"? I think the WebIDL concept is useful here for those who may not be familiar with the topic.

Reply:
I think the agenda item should link to the relevant technical discussions to read beforehand which are then summarized during the discussion, not capture parts of the technical discussion within the agenda item itself, since the same unfamiliarity will likely conflate "`DOMString` semantics" with "two byte encodings".